forgerock-auth/src/main/java/org/forgerock/android/auth/FRUser.java

```diff
@@ -317,35 +305,38 @@ private void login(Context context, FragmentManager manager, FRListener<FRUser>
             }
 
             try {
-                validateRedirectUri(context);
+                validateRedirectUri(activity.getApplicationContext());
```

Comment:
i think we can just give validateRedirectUri(activity)

Reply:
Updated.